components/divider/metadata/divider.yml

```diff
@@ -4,64 +4,125 @@ SpectrumSiteSlug: https://spectrum.adobe.com/page/divider/
+                <svg class="spectrum-Icon spectrum-Icon--sizeM spectrum-ActionButton-icon" focusable="false" aria-hidden="true" aria-label="Select">
+                  <use xlink:href="#spectrum-icon-18-Edit" />
+                </svg>
+              </button>
+            </div>
+          </div>
+
+          <div class="spectrum-Examples-item">
+            <h4 class="spectrum-Heading spectrum-Heading--sizeXS spectrum-Examples-itemHeading" style="margin-bottom: 2rem">Medium</h4>
+            <div class="spectrum-ActionGroup spectrum-ActionGroup--horizontal">
+              <button class="spectrum-ActionButton spectrum-ActionButton--sizeM spectrum-ActionButton--quiet">
+                <svg class="spectrum-Icon spectrum-Icon--sizeM spectrum-ActionButton-icon" focusable="false" aria-hidden="true" aria-label="Properties">
+                  <use xlink:href="#spectrum-icon-18-Properties" />
+                </svg>
+              </button>
+              <div class="spectrum-Divider spectrum-Divider--sizeM spectrum-Divider--vertical" style="height: auto; align-self: stretch;"></div>
+              <button class="spectrum-ActionButton spectrum-ActionButton--sizeM spectrum-ActionButton--quiet">
+                <svg class="spectrum-Icon spectrum-Icon--sizeM spectrum-ActionButton-icon" focusable="false" aria-hidden="true" aria-label="Select">
+                  <use xlink:href="#spectrum-icon-18-Edit" />
+                </svg>
+              </button>
+            </div>
+          </div>
+
+          <div class="spectrum-Examples-item">
+            <h4 class="spectrum-Heading spectrum-Heading--sizeXS spectrum-Examples-itemHeading" style="margin-bottom: 2rem">Large</h4>
+            <div class="spectrum-ActionGroup spectrum-ActionGroup--horizontal">
+              <button class="spectrum-ActionButton spectrum-ActionButton--sizeM spectrum-ActionButton--quiet">
+                <svg class="spectrum-Icon spectrum-Icon--sizeM spectrum-ActionButton-icon" focusable="false" aria-hidden="true" aria-label="Properties">
+                  <use xlink:href="#spectrum-icon-18-Properties" />
+                </svg>
+              </button>
+              <div class="spectrum-Divider spectrum-Divider--sizeL spectrum-Divider--vertical" style="height: auto; align-self: stretch;"></div>
+              <button class="spectrum-ActionButton spectrum-ActionButton--sizeM spectrum-ActionButton--quiet">
+                <svg class="spectrum-Icon spectrum-Icon--sizeM spectrum-ActionButton-icon" focusable="false" aria-hidden="true" aria-label="Select">
+                  <use xlink:href="#spectrum-icon-18-Edit" />
+                </svg>
+              </button>
+            </div>
+          </div>
+
+        </div>
+
+
+  - id: divider-static-white
+    name: Static White
+    markup: |
+      <div class="spectrum-Examples">
+        <div style="background-color: rgb(0, 119, 114); color: rgb(255, 255, 255); padding: 15px 20px; width: 100%;">
```

Comment:
Do we have a general class we could apply for the static-white / black docs instead of manually in-line styling it?
Main reason here is that we once in a while change those background-color for improved contrast ratios or the latest color trends...

Reply:
I referenced actionbutton for how I approached this, and found that the static white background color was applied with inline rgb styling. I changed the values to reflect the XD redline colors. 

I think it would be great to have a class we could apply to the examples wrapper so the value can more easily be changed. It looks like they should be added in components/site, which contains styles for the examples. Are we going to be migrating the site component?

Reply:
Great suggestion, @bernhard-adobe, but I think this is fine inline for now. We'll probably end up refreshing the Docs at some point, so let's just leave this as is.

Reply:
I keep adding in my Switch work to `components/tokens/custom-spectrum/custom-vars.css`.
Approving for now.